<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="Dip_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_7">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Dip_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
